.github/workflows/test-bbtools.yml

```diff
@@ -1,15 +1,29 @@
 name: Test BBTools image
 
+env:
+  TOOL_NAME: bbtools
+  TOOL_VERSION: "38.96"
+
 on:
   workflow_dispatch:
   pull_request:
     paths:
-      - 'bbtools/38.96/Dockerfile'
+      - 'bbtools/38.96/Dockerfile' # cannot use env vars
```

Comment:
Does this still work with 
```
- 'bbtools/*/Dockerfile'
```

Reply:
As far as I know, that would still work, but it would apply to _all_ versions of the tool.